### PR TITLE
Added support for create-resend campaign action.

### PR DESCRIPTION
**Pull request by michaelwalkerfl**

### CODE REVIEW

File: mailchimp3/entities/campaignactions.py [hunk 1]
```diff
@@ -115,6 +115,17 @@ def send(self, campaign_id):
         return self._mc_client._post(url=self._build_path(campaign_id, 'actions/send'))
 
 
+    def resend(self, campaign_id):
+        """
+        Resend a MailChimp campaign to those that did not open previously.
```
Comment (stephenross) [inline mailchimp3/entities/campaignactions.py]: Documentation: Please replace this description with the one provided by the API docs. `Creates a Resend to Non-Openers version of this campaign. We will also check if this campaign meets the criteria for Resend to Non-Openers campaigns.`

Comment (michaelwalkerfl) [reply mailchimp3/entities/campaignactions.py]: Thank you for the feedback! Per your instructions, I've updated the description to match the API docs and updated the README files and setup.py to reflect Python version 3.0.13.

File: README.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-|mailchimp3 v3.0.12 on PyPi| |MIT license| |Stable|
+|mailchimp3 v3.0.13 on PyPi| |MIT license| |Stable|
```
Comment (stephenross) [inline README.rst]: The link here is defined at the bottom and needs to be updated down there as well

Comment (michaelwalkerfl) [reply README.rst]: Yikes, sorry about that. This has been updated. Thanks again!

**Comment by stephenross** [comment]
@charlesthk New version ready to go up 3.0.13

**Comment by charlesthk** [comment]
@stephenross thanks, 3.0.13 is now on PyPi.

**Pull request merged**